Branch-SDK/src/io/branch/referral/ServerRequest.java

```diff
@@ -385,7 +385,7 @@ private void updateGAdsParams() {
                 if (version == BRANCH_API_VERSION.V2) {
                     JSONObject userDataObj = params_.optJSONObject(Defines.Jsonkey.UserData.getKey());
                     if (userDataObj != null) {
-                        userDataObj.put(Defines.Jsonkey.GoogleAdvertisingID.getKey(), systemObserver_.GAIDString_);
+                        userDataObj.put(Defines.Jsonkey.AAID.getKey(), systemObserver_.GAIDString_);
```

Comment:
This whole AAID -> GAID drives me a bit crazy.

v1/open sends `google_advertising_id=REAL_GAID`
v1/install sends `google_advertising_id=REAL_GAID`
v2/event sends `aaid=REAL_GAID`

With this new change,
v1/event will send `google_advertising_id=REAL_GAID`

Reply:
even better if we could update gateway to look for `google_advertising_id` for v2/event too :D

Reply:
Brian S is okay with  v1/event with google_advertising_id=REAL_GAID and updating backend to accept.